ci/ray_ci/doc/update_cache_env.py

```diff
@@ -0,0 +1,89 @@
+import pickle
+from sphinx.project import Project
+import os
+import time
+from typing import List
+from datetime import datetime, timezone
+import click
+
+
+def list_pending_files(ray_dir: str) -> List[str]:
+    """List all files that are added/modified in git repo."""
+    pending_files = []
+    with open(f"{ray_dir}/pending_files.txt", "r") as f:
```

Comment:
where is this txt file coming from? also maybe put the `pending_files.txt` into a constant

Reply:
done! It's coming from the load doc cache step (in the other PR)